test/controllers/welcome_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
```

Comment:
We do not need this controller tests. Controller are covered by cucumber tests ( Feature tests)


This does not mean that controller tests are not important, it depends on the organisation. Some prefer and insist of these tests but as long as we can cover as much with `end to end tests` then IMO we do not need it